CMakeLists.txt

```diff
@@ -150,6 +150,10 @@ ocv_cmake_eval(DEBUG_PRE ONCE)
 ocv_clear_vars(OpenCVModules_TARGETS)
 
 include(cmake/OpenCVDownload.cmake)
+# ----------------------------------------------------------------------------
+#  Initialize download in case mirror is used
+# ----------------------------------------------------------------------------
+ocv_init_download()
```

Comment:
Move this call to the end of `cmake/OpenCVDownload.cmake`. No need to call it separately.

Reply:
Done.